apps/fz_http/lib/fz_http/telemetry.ex

```diff
@@ -162,9 +162,7 @@ defmodule FzHttp.Telemetry do
   end
 
   defp outbound_email? do
-    from_email = FzHttp.Config.fetch_env!(:fz_http, FzHttpWeb.Mailer)[:from_email]
-
-    !is_nil(from_email)
+    not is_nil(FzHttp.Config.fetch_env!(:fz_http, FzHttpWeb.Mailer)[:from_email])
```

Comment:
`: from_email ` can be set, but it doesn't mean that other configs (provider-based and adapter) are set too..

Reply:
BTW this check is a copy of `FzHttpWeb.Mailer.active?/0`

Reply:
So `from_email` and `adapter` are the only ones required to send outbound email. We set the adapter in the `config/*.exs` files and [document using `from_email` to enable/disable outbound email](https://docs.firezone.dev/reference/env-vars/), but I agree this is more clear if we check for both here.

Reply:
Updated.